server/api/controllers/forms/post-create-form.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  friendlyName: "Create Form",
+
+  description: "Create a new form",
+
+  inputs: {
+    formToSave: {
+      type: "json",
+      required: true,
+      description: "form saved object",
+    },
+  },
+
+  exits: {
+    success: {
+      description: "New form was created successfully.",
+    },
+    error: {
+      description: "Failed to create new form.",
+    },
+  },
+
+  fn: async function ({ formToSave }, exits) {
+    try {
+      const newForm = await Form.create(formToSave).fetch();
```

Comment:
hmm, why do we need `.fetch()` if we are not using newForm for anything else afterwards anyway?